pkg/syncer/cnsoperator/controller/cnsnodevmattachment/cnsnodevmattachment_controller.go

```diff
@@ -727,17 +730,23 @@ func updateSVPVC(ctx context.Context, client client.Client,
 			}
 			err := client.Update(ctx, latestPVCObject)
 			if err != nil {
+				if apierrors.IsConflict(err) {
+					log.Infof("Observed conflict again, while updating the SV PVC %q in namespace %q."+
+						"Returning error and setting the faulttype as nonStorage fault as the next reconciliation "+
+						"will be invoked.", pvc.Name, pvc.Namespace)
+					return csifault.CSIResourceUpdateConflictFault, err
+				}
 				log.Errorf("failed to update SV PVC : %q on namespace: %q. Error: %+v",
```

Comment:
This log should also be in an `else` block when error is other than Conflict error. And it should return `csifault.CSIApiServerOperationFault` fault type.

Reply:
Right, looks like the return codes were incorrectly defined. Addressed